bin/study.js

```diff
@@ -1,5 +1,13 @@
+'use strict';
+
 // Repeat Ruby Lab for JS
+
 // let myArray = [1, 2, 3, 4, 5];
+let newArr = Array.new[10];
 
+for( let i = 0; i < 10; i++ )
+{
+  newArr[i] = i * 2;
+}
```

Comment:
take a look at this solution: `let arrayTimesTwo = new Array(10).fill(1).map((e, i) => i *2);`